src/style.css

```diff
@@ -0,0 +1,124 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url(./fonts/Avenir-Book.ttf);
+}
```

Comment:
Please check this [guide](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) (7 point)